<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fm_Gvendas</class>
 <widget class="QDialog" name="Fm_Gvendas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateEdit" name="Date_begin">
       <property name="currentSection">
        <enum>QDateTimeEdit::YearSection</enum>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="Date_end">
       <property name="currentSection">
        <enum>QDateTimeEdit::YearSection</enum>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Bt_search">
       <property name="text">
        <string>Filtrar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BT_show">
       <property name="text">
        <string>Mostrar Vendas</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="TableView_vendas"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTableWidget" name="TableView_produtos"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,2">
     <item>
      <widget class="QPushButton" name="BT_pdf">
       <property name="text">
        <string>Gerar PDF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Bt_exel">
       <property name="text">
        <string>Gerar Planilha</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="teste">
       <property name="text">
        <string>Ler dados da planilha</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
